PhotoMap/Features/Map/Map.storyboard

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please exclude this file from PR. There's no actual changes in the file. It's just Xcode preview chat changed rect frame as you can see. E.g. 
`<rect key="frame" x="0.0" y="0.0" width="600" height="600"/>`

Reply:
Unresolved

Reply:
Still there :)